### PR TITLE
docs(v2): showcase user Tasit

### PR DESCRIPTION
**Pull request by pcowgill**
### Motivation

Adding another example of a Docusaurus v2 user to the showcase.

[docs.tasit.io](https://docs.tasit.io)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Please confirm that the showcase page renders properly with the new addition. Thanks!

## Related PRs

n/a

### CODE REVIEW

**Comment by pcowgill** [comment]
I'm not sure why the `prettier` check failed. This was the output:

```
#!/bin/bash -eo pipefail 
yarn prettier:diff 
yarn run v1.22.4
$ prettier --config .prettierrc --list-different "**/*.{js,ts}"
website/src/data/users.js
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Exited with code exit status 1 
CircleCI received exit code 1 
```

**Comment by pcowgill** [comment]
FYI I signed the CLA just now.

File: website/src/data/users.js [hunk 1]
```diff
@@ -155,6 +155,15 @@ const users = [
     fbOpenSource: false,
     pinned: false,
   },
+  {
+    title: 'Tasit',
+    description: 'A JavaScript SDK for building native mobile Ethereum dapps with React Native',
```
Comment (yangshun) [inline website/src/data/users.js]: Prettier failed because this line is too long and should be on the next.

**Comment by yangshun** [comment]
Thanks!

File: website/src/data/users.js (hunk 1 above)
Comment (pcowgill) [reply website/src/data/users.js]: Ah ok, makes sense. Thanks!

**Pull request merged**